aws-iot-jobtemplate/resource-role.yaml

```diff
@@ -23,10 +23,10 @@ Resources:
             Statement:
               - Effect: Allow
                 Action:
-                - "iot:CreateJobTemplates"
-                - "iot:DeleteJobTemplates"
+                - "iot:CreateJobTemplate"
+                - "iot:DeleteJobTemplate"
                 - "iot:DescribeJobTemplates"
-                - "iot:ListJobTemplates"
+                - "iot:ListJobTemplate"
```

Comment:
Action name should be `ListJobTemplates` and `DescribeJobTemplate`.

Reply:
Yikes, thanks for noticing that.